### DISCUSSION TITLE
Support for the new Roborock integration

### DISCUSSION THREAD

**Discussion by bellini666** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#431] [Ideas]
Home assistant just released a new integration that uses the Roborock app directly instead of the xiomi one: https://www.home-assistant.io/integrations/roborock

**Comment by PiotrMachowski** [comment]
Yes, it has. But what does it has in common with this integration? There already is a [feature request](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues/388) to add support for Roborock platform to Map Extractor, a [custom integration](https://github.com/humbertogontijo/homeassistant-roborock) that supports retrieving maps, also it is planned to add support for maps to the official integration as well.
